didact/camelk/first-integration.md

```diff
@@ -0,0 +1,109 @@
+
+## Prerequisites 
+
+You must have a few things set up prior to walking through the steps in this tutorial. 
+
+<a href='didact://?commandId=vscode.didact.validateAllRequirements' title='Validate all requirements!'><button>Validate all Requirements at Once!</button></a>
+
+| Requirement (Click to Verify)  | Availability | Additional Information/Solution |
+| :--- | :--- | :--- |
+| [At least one folder exists in the workspace](didact://?commandId=vscode.didact.workspaceFolderExistsCheck&text=workspace-folder-status&completion=A%20valid%20folder%20exists%20in%20the%20workspace. "Ensure that at least one folder exists in the user workspace"){.didact} | *Status: unknown*{#workspace-folder-status} | Create a workspace folder (or [click here to create a temporary folder](didact://?commandId=vscode.didact.createWorkspaceFolder&completion=Created%20temporary%20folder%20in%20the%20workspace. "Create a temporary folder and add it to the workspace."){.didact}), close, and reopen the Didact window
+| [Minikube is accessible and running at the command line](didact://?commandId=vscode.didact.requirementCheck&text=minikube-requirements-status$$minikube%20status$$host:%20Running&completion=Minikube%20is%20available%20on%20this%20system. "Tests to see if `minikube status` returns a result"){.didact} 	| *Status: unknown*{#minikube-requirements-status} 	| See [Installing Camel K on Minikube](https://camel.apache.org/camel-k/latest/installation/minikube.html "Documentation on how to Install Apache Camel K on Minikube, with links to the official doc for Minikube installation")
+| [VS Code Extension Pack for Apache Camel by Red Hat is installed](didact://?commandId=vscode.didact.extensionRequirementCheck&text=extension-requirement-status$$redhat.apache-camel-extension-pack&completion=Camel%20extension%20pack%20available. "Checks the VS Code workspace to make sure the extension pack is installed"){.didact} | *Status: unknown*{#extension-requirement-status} 	| [Click here to install](vscode:extension/redhat.apache-camel-extension-pack "Opens the extension page and provides an install link") |
+| [VS Code Tooling for Apache Camel K by Red Hat is installed](didact://?commandId=vscode.didact.extensionRequirementCheck&text=camelk-extension-requirement-status$$redhat.vscode-camelk&completion=Camel%20K%20extension%20pack%20available. "Checks the VS Code workspace to make sure the extension pack is installed"){.didact} | *Status: unknown*{#camelk-extension-requirement-status} 	| [Click here to install](vscode:extension/redhat.vscode-camelk "Opens the extension page and provides an install link") |
+
+## Your First Camel K Integration
+
+You can write an integration in one of several languages supported ([Groovy, Kotlin, JavaScript, Java, XML, etc.](https://camel.apache.org/camel-k/latest/languages/languages.html)), but today we're going to focus on Groovy which is one of the easiest and most popular.
```

Comment:
I know the tutorial was written before it was decided to have Groovy in most focused languages.
Just wondering if we should not shift to another one in the first tutorial that we are providing.

Reply:
I don't think we stricly need to change what we have. We can add more eventually, but I don't see reasons to change the current one.

Reply:
a reason would be to provide a tutorial on one of the language that Fuse is focusing/planned to support first (Java, xml, Javascript).

i'm fine to keep Groovy, it is just that we are not in phase of Fuse priorities.

Reply:
Easy enough to change languages, but I was going with what was suggested by the runtime we're supporting and simply migrating the suggested Camel K runtime example from the documentation https://camel.apache.org/camel-k/latest/running/running.html 

In the long run, I anticipate that these files will be written/modified by our documentation team, which means we will have less direct input on their content.